### ISSUE TITLE
Current kernel 6.1.102-3.ph5 is missing nft_counter module

### ISSUE THREAD

**Issue by jnovak-netsystemcz** [vmware/photon#1564]
### Describe the bug

There is no nft_counter module in kernel package even other nft_* modules are there. Therefore counters can't be used in firewall rules.

### Reproduction steps

1. Check content of `/lib/modules/6.1.102-3.ph5/kernel/net/netfilter/`, the file is missing
2. `nft -f fw_rule.conf` shows error

### Expected behavior

`nft -f fw_rule.conf` should load the rule.

### Additional context

Content of `fw_rule.conf`:

```
table inet t_test {

  set testlbl { type ipv4_addr; }

  counter cnt_test {
    comment "Test counter"
  }

  chain ch_test {
    ip saddr @testlbl counter name cnt_test
  }

}
```

**Comment by dcasota** [comment]
@sshedi I think @jnovak-netsystemcz is right.

Tested on x86_64 Ph5/Kernel6.1.106-esx@VMWorkstation and on x86_64 Ph5/Kernel6.6.36-generic@WSL2
![image](https://github.com/user-attachments/assets/498a58b7-b633-4c62-808e-f1348bda30cc)

**Comment by jaankit** [comment]
working on it. Thanks for reporting.

**Comment by tapakund** [comment]
Fixed with https://github.com/vmware/photon/commit/56de87da04f41d2ae513e3eaccb146cef456ddc9

**Comment by jnovak-netsystemcz** [comment]
Thank you

**Comment by jnovak-netsystemcz** [comment]
Hi. I tested it on 6.1.106-4.ph5 and it is still same. Should the patch be there or in kernel one version later?

Should it create kernel module nft_objref? If so, the file is missing in package.

**Comment by dcasota** [comment]
@jnovak-netsystemcz With respect to https://github.com/vmware/photon/commit/56de87da04f41d2ae513e3eaccb146cef456ddc9 it should be fixed with 6.1.106-5.ph5.

It has been published a few of hours ago. Might have been a time glitch.

![image](https://github.com/user-attachments/assets/e6604127-c652-4b63-8a85-ebe8fdf7b7f6)

**Comment by jnovak-netsystemcz** [comment]
I retested it after update of kernel and it works now. Thank you.
